packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -32,6 +35,62 @@ sapien, et dapibus mi aliquet non. Pellentesque auctor sagittis lectus vitae rho
 ante in, vestibulum nulla.`;
 const richMessage = <span style={{ fontWeight: 700 }}>Successful alert with custom rendered message</span>;
 
+const topBannerAlertStyle = {};
+const bannerAlertStyle = { marginTop: spacingLg };
```

Comment:
Can we be a bit smarter about these styles? I.e. separate out to separate `.scss` stylesheet and prevent the style from being applied on the first banner alert (i.e. `:not(:first)`)